README.md

```diff
@@ -14,35 +14,49 @@ The easiest way to install `node.js` is with [nave.sh](https://github.com/isaacs
 ## Installation
 
 ```bash
-$ npm install pelias-openstreetmap-polygons
+    $ npm install pelias-openstreetmap-polygons
```

Comment:
Is there a reason these are indented? Doesn't really matter, but I haven't seen it before and it's a bit inconsistent with our other READMEs.